EventDisplay/src/DataInterface.cc

```diff
@@ -777,6 +777,38 @@ void DataInterface::findBoundaryP(spaceminmax &m, double x, double y, double z)
   if(std::isnan(m.maxz) || z>m.maxz) m.maxz=z;
 }
 
+using LHPT = KinKal::PiecewiseTrajectory<KinKal::LoopHelix>;
+using CHPT = KinKal::PiecewiseTrajectory<KinKal::CentralHelix>;
+using KLPT = KinKal::PiecewiseTrajectory<KinKal::KinematicLine>;
+template<class KTRAJ> void DataInterface::fillKalSeedTrajectory(std::unique_ptr<KTRAJ> &trajectory,
+                                                                int particleid, int trackclass, int trackclassindex, double p1,
+                                                                boost::shared_ptr<ComponentInfo> info)
+{
+  double t1=trajectory->range().begin();
+  double t2=trajectory->range().end();
+
+  double x1=trajectory->position3(t1).x();
+  double y1=trajectory->position3(t1).y();
+  double z1=trajectory->position3(t1).z();
+  double x2=trajectory->position3(t2).x();
+  double y2=trajectory->position3(t2).y();
+  double z2=trajectory->position3(t2).z();
+
+  boost::shared_ptr<Track> track(new Track(x1,y1,z1,t1, x2,y2,z2,t2,
+                                           particleid, trackclass, trackclassindex, p1,
+                                           _geometrymanager, _topvolume, _mainframe, info, false));
+  _components.push_back(track);
+  _tracks.push_back(track);
+
+  for(double t=t1; t<=t2; t+=0.1)
```

Comment:
Please make the time step a configurable parameter

Reply:
Done.